### ISSUE TITLE
Possible Time.timeScale double count

### ISSUE THREAD

**Issue by selenexwu** [5FiftySix6/HollowKnight.Lightbringer#2]
I was reading through the code because I'm trying to get into HK modding and was curious how your mod worked, and I noticed that you multiply Time.deltaTime by Time.timeScale [here](https://github.com/fifty-six/HollowKnight.Lightbringer/blob/cce279797840f922f36d082927ba8d181a68c426/Lightbringer/Lightbringer.cs#L610). I have some Unity background and I double checked the docs and I'm fairly sure Time.deltaTime is already multiplied by Time.timeScale internally. (Which is why Time.unscaledDeltaTime exists.) Its entirely possible theres a subtlety that I'm missing here but I thought I'd point it out in case there isn't.

**Comment by fifty-six** [comment]
Yeah, that seems right - I'll fix it whenever I get around to porting this to 1.5 if 753 doesn't first
